### PR TITLE
OLH-902 : Update Logging to exclude logging personal data from txma events

### PR DESCRIPTION
**Pull request by divyaparameswaran**
Update lambdas to not log any data from the TXMA event.

### CODE REVIEW

File: lambda/query-activity-log/query-activity-log.ts [hunk 1]
```diff
@@ -60,9 +60,7 @@ export const validateTxmaEventBody = (txmaEvent: TxmaEvent): void => {
   ) {
     validateUser(txmaEvent.user);
   } else {
-    throw new Error(
-      `Could not validate UserServices ${JSON.stringify(txmaEvent)}`
```
Comment (DougsCER) [inline lambda/query-activity-log/query-activity-log.ts:64]: non PII data from the TXMA event could still be helpful to triage issues. eg the client_id, event_name.
Could this be logged?

Comment (divyaparameswaran) [reply lambda/query-activity-log/query-activity-log.ts:64]: So these messages go to the DLQ and we log the message ID there. We can get the entire paylod if we look at the message in the DLQ ? WOuld it be helpful to still log client_id and event_name

Comment (divyaparameswaran) [reply lambda/query-activity-log/query-activity-log.ts:64]: Ah sorry we don't log message Id for messages sent to DLQ maybe that would be a better option ?

Comment (DougsCER) [reply lambda/query-activity-log/query-activity-log.ts:64]: yeah, message ID seems key.  If the error log gives us message ID and we can see the message content from DLQ then that's grand

File: lambda/delete-activity-log/delete-activity-log.ts [hunk 2]
```diff
@@ -133,8 +131,10 @@ export const handler = async (event: SNSEvent): Promise<void> => {
           QueueUrl: DLQ_URL,
           MessageBody: record.Sns.Message,
         };
-        console.log("sending message to SQS %s", JSON.stringify(message));
-        await sqsClient.send(new SendMessageCommand(message));
+        const result = await sqsClient.send(new SendMessageCommand(message));
+        console.log(
+          `[Message sent to DLQ] with message id = ${result.MessageId}`
```
[the remark this answers is not in the document]
Comment (divyaparameswaran) [reply lambda/delete-activity-log/delete-activity-log.ts:136]: Yes it still logs Validation Error i.e Could not validate User. Then we can check the message to see the payload to fifure out what the missing piece was ?

**Review by DougsCER — APPROVED** [review]

File: lambda/delete-activity-log/delete-activity-log.ts (hunk 2 above)
[the remark this answers is not in the document]
Comment (DougsCER) [reply lambda/delete-activity-log/delete-activity-log.ts:136]: Chatted to Divya on call. 
She pointed out this is being logged by a log line above that's not shown in this PR summary. 

I've approved on the basis that the log lines are moved to be next to eachother

**Pull request merged**